packages/iris-grid/src/EmptyIrisGridModel.ts

```diff
@@ -1,42 +1,21 @@
 /* eslint class-methods-use-this: "off" */
-import {
-  GridRange,
-  ModelIndex,
-  MoveOperation,
-  VisibleIndex,
-} from '@deephaven/grid';
+import { ModelIndex, MoveOperation } from '@deephaven/grid';
 import { dh as DhType } from '@deephaven/jsapi-types';
 import { ColumnName, Formatter } from '@deephaven/jsapi-utils';
 import { EMPTY_ARRAY, EMPTY_MAP } from '@deephaven/utils';
-import IrisGridModel from './IrisGridModel';
 import ColumnHeaderGroup from './ColumnHeaderGroup';
-import {
-  PendingDataErrorMap,
-  PendingDataMap,
-  UITotalsTableConfig,
-} from './CommonTypes';
+import { UITotalsTableConfig } from './CommonTypes';
+import IrisGridSchemaModelTemplate from './IrisGridSchemaModelTemplate';
 
-class EmptyIrisGridModel extends IrisGridModel {
+class EmptyIrisGridModel extends IrisGridSchemaModelTemplate {
   constructor(dh: typeof DhType, formatter = new Formatter(dh)) {
-    super(dh);
-
-    this.modelFormatter = formatter;
-  }
-
-  modelFormatter: Formatter;
-
-  get rowCount(): number {
-    return 0;
+    super(dh, []);
```

Comment:
Need to pass the `formatter` through.
```suggestion
    super(dh, [], formatter);
```

That being said, it's not like this really _needs_ a formatter since it's an empty grid, but :shrug: